### PR TITLE
feat(website): update node running guides

### PR DESCRIPTION
**Pull request by 2manslkh**

### CODE REVIEW

File: packages/website/pages/docs/guides/run-a-node/enable-a-proposer.mdx [hunk 1]
```diff
@@ -59,7 +59,15 @@ Click `Write` and confirm the transaction in your wallet.
 
 ### Open .env file in `simple-taiko-node`
 
-Open the `.env` (or `.env.l3` for Eldfell L3 proposers) file in the `simple-taiko-node` directory.
+For Grimsvotn L2:
+```sh
+nano .env
+```
+
+For Eldfell L3:
```
Comment (d1onys1us) [inline packages/website/pages/docs/guides/run-a-node/enable-a-proposer.mdx:67]:
```suggestion
For Eldfell L3:

```

File: packages/website/pages/docs/guides/run-a-node/enable-a-prover.mdx [hunk 2]
```diff
@@ -28,7 +28,17 @@ This guide will help you enable your Taiko node as a prover. Read more about pro
 
 <Steps>
 ### Open the .env file in `simple-taiko-node`
-Open the `.env` (or `.env.l3` for Eldfell L3 provers) file in the `simple-taiko-node` directory.
+
+For Grimsvotn L2:
```
Comment (d1onys1us) [inline packages/website/pages/docs/guides/run-a-node/enable-a-prover.mdx:32]:
```suggestion
For Grimsvotn L2:

```

File: packages/website/pages/docs/guides/run-a-node/enable-a-prover.mdx [hunk 3]
```diff
@@ -28,7 +28,17 @@ This guide will help you enable your Taiko node as a prover. Read more about pro
...
+```sh
+nano .env
+```
+
+For Eldfell L3:
```
Comment (d1onys1us) [inline packages/website/pages/docs/guides/run-a-node/enable-a-prover.mdx:37]:
```suggestion
For Eldfell L3:

```

**Review by dionysuzx — DISMISSED** [review]

**Review by dantaik — APPROVED** [review]

**Review by dionysuzx — APPROVED** [review]

**Review by dionysuzx — APPROVED** [review]

**Comment by d1onys1us** [comment]
> link.I was the one who came up with the idea. #14398 I want a POAP...

@gitpoap-bot please award @tukimi666 with a poap

**Pull request merged**